amgut/templates/new_participant.html

```diff
@@ -10,7 +87,7 @@
     }
 
     function relax_parents(box) {
-        var sec_parent = document.getElementById(box.form.id + "_parent_2_name");
+        var sec_parent = $("#" + box.form.id + "_parent_2_name");
```

Comment:
uses jquery instead of standard javascript. Was done to fix an issue with safari, but was not actually fixed with this, so can revert if wanted. Works either way though.